ctapipe/image/tests/test_waveform_cleaning.py

```diff
@@ -39,8 +39,24 @@ def test_checm_cleaner():
     data_ped = data - np.atleast_3d(ped/nsamples)
     data_ped = np.array([data_ped[0], data_ped[0]])  # Test LG functionality
 
-    cleaner = CHECMWaveformCleaner(None, None)
+    cleaner = CHECMWaveformCleanerAverage(None, None)
     cleaned = cleaner.apply(data_ped)
 
     assert_almost_equal(data_ped[0, 0, 0], -2.8, 1)
     assert_almost_equal(cleaned[0, 0, 0], -6.4, 1)
+
+
+def test_checm_cleaner_local():
+    telid = 11
+    event = get_test_event()
```

Comment:
at some point we should define the test event as a global test fixture, so it is onyl read once (that will speed up the tests, since so many are opening and reading simtel files now). Then you just change your function to be `def test_checm_cleaner_local(test_event)`.  That can be done later though.